src/utils/defaultLanguageStrings.ts

```diff
@@ -290,6 +291,13 @@ export default {
   REVIEW_OVERVIEW_STAGE: 'Stage',
   REVIEW_OVERVIEW_OUTCOME: 'Outcome',
   REVIEW_OVERVIEW_SERIAL: 'Serial',
+  REVIEW_OVERVIEW_DEADLINE: 'Applicant Deadline',
+  REVIEW_OVERVIEW_EXTEND_BY: 'Extend deadline by',
+  REVIEW_OVERVIEW_DAYS: 'days',
```

Comment:
It's possibly problematic to split the sentence like this -- other languages might structure it differently. But it's not really feasible to put a substitution in when the substitution needs to be a Form element (number selector).